lib/jobs/run-sku-graph.js

```diff
@@ -41,7 +41,7 @@ function runSkuGraphJobFactory(
         RunSkuGraphJob.super_.call(this, logger, options, context, taskId);
 
         this.nodeId = this.options.nodeId;
-        assert.isMongoId(this.nodeId);
+        assert.string(this.nodeId);
```

Comment:
Any thoughts on monkey patching in something like an assert.nodeid () method? May be nice to assert on something more specific than just a string